### DISCUSSION TITLE
Listen on IPv6 and IPv4 simultaneously

### DISCUSSION THREAD

**Discussion by havok4u** [poem-web/poem#631] [General]
Does Poem have the ability to listen on ipv6 and ipv4 at the same time?  If so, how does one accomplish that.  I can listen on ipv4 or ipv6, just cannot figure out how to listen on both.  I don't see any queries on that here or through Google.  Thanks in advance.

**Comment by sunli829** [comment]
You can combine multiple listeners.

https://github.com/poem-web/poem/tree/master/examples/poem/combined-listeners
